### PR TITLE
encode image data to JPEG

### PR DESCRIPTION
**Pull request by isabellehuangg**

### CODE REVIEW

File: lte/modules/image_encode.py [hunk 1]
```diff
@@ -0,0 +1,17 @@
+"""
+Encodes image from original format to JPEG, filtering out A channel if PNG
+"""
+
+import io
+
+from PIL import Image
+
+def encode(png_image):
```
Comment (mgupta27) [inline lte/modules/image_encode.py:9]: This function does not specify the parameter type or the type that it returns. Can you please add that.

Also, the image that is being passed in will be in the format of a numpy array in RGB format, not a PNG image.

**Review by mgupta27 — COMMENTED** [review]
Reviewed.

File: lte/modules/image_encode.py (hunk 1 above)
Comment (isabellehuangg) [reply lte/modules/image_encode.py]: Added

File: lte/modules/image_encode.py [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+import io
+
+from PIL import Image
+import numpy as np
+
+def encode(image_array: np.ndarray) -> "io.BytesIO | bytes":
+    img = Image.fromarray(image_array)
+
+    buffer = io.BytesIO()
+    img.save(buffer, format = 'JPEG', quality=100)
```
Comment (TongguangZhang) [inline lte/modules/image_encode.py:14]: let's make the format and quality params constants

**Review by TongguangZhang — COMMENTED** [review]
Reviewed, make sure you run the formatter with `black .` in the terminal to pass the linter (make sure you've installed the dependencies
Could you also write the unit tests for encode?

**Review by mgupta27 — APPROVED** [review]
LGTM

**Pull request merged**